setup.py

```diff
@@ -3,6 +3,7 @@
 
 install_requires = [
     "wagtail>=2.3,<2.10",
```

Comment:
```suggestion
    "wagtail>=2.3,<2.11",
```

since we're running on 2.10rc1